nodebuilder/tests/swamp/swamp.go

```diff
@@ -324,18 +324,26 @@ func (s *Swamp) remove(rn *nodebuilder.Node, sn []*nodebuilder.Node) ([]*nodebui
 }
 
 // Connect allows to connect peers after hard disconnection.
-func (s *Swamp) Connect(t *testing.T, peerA, peerB peer.ID) {
-	_, err := s.Network.LinkPeers(peerA, peerB)
+func (s *Swamp) Connect(t *testing.T, peerA, peerB *nodebuilder.Node) {
+	_, err := s.Network.LinkPeers(peerA.Host.ID(), peerB.Host.ID())
 	require.NoError(t, err)
-	_, err = s.Network.ConnectPeers(peerA, peerB)
+	_, err = s.Network.ConnectPeers(peerA.Host.ID(), peerB.Host.ID())
 	require.NoError(t, err)
 }
 
 // Disconnect allows to break a connection between two peers without any possibility to
 // re-establish it. Order is very important here. We have to unlink peers first, and only after
 // that call disconnect. This is hard disconnect and peers will not be able to reconnect.
 // In order to reconnect peers again, please use swamp.Connect
-func (s *Swamp) Disconnect(t *testing.T, peerA, peerB peer.ID) {
-	require.NoError(t, s.Network.UnlinkPeers(peerA, peerB))
-	require.NoError(t, s.Network.DisconnectPeers(peerA, peerB))
+func (s *Swamp) Disconnect(t *testing.T, peerA, peerB *nodebuilder.Node) {
+	require.NoError(t, s.Network.UnlinkPeers(peerA.Host.ID(), peerB.Host.ID()))
+	require.NoError(t, s.Network.DisconnectPeers(peerA.Host.ID(), peerB.Host.ID()))
+}
+
+func WithTrustedPeers(t *testing.T, cfg *nodebuilder.Config, trustedPeers ...*nodebuilder.Node) {
+	for _, trusted := range trustedPeers {
+		addrs, err := peer.AddrInfoToP2pAddrs(host.InfoFromHost(trusted.Host))
+		require.NoError(t, err)
+		cfg.Header.TrustedPeers = append(cfg.Header.TrustedPeers, addrs[0].String())
+	}
 }
```

Comment:
Let's set trusted or bootstrap peers for nodes automatically. We should still allow users to set trusted peers if their test case requires it, but that should not be necessary to startup a healthy node.

Reply:
https://github.com/celestiaorg/celestia-node/pull/2162/commits/2883a67bb299e5c5d4aea5e9aef4cd131d6d6698